sql/hive/v0.13.1/src/main/scala/org/apache/spark/sql/hive/Shim13.scala

```diff
@@ -297,7 +297,7 @@ private[hive] object HiveShim {
   def getStatsSetupConstRawDataSize = StatsSetupConst.RAW_DATA_SIZE
 
   def createDefaultDBIfNeeded(context: HiveContext) = {
-    context.runSqlHive("CREATE DATABASE default")
+    context.runSqlHive("CREATE DATABASE IF NOT EXISTS default")
```

Comment:
LTGM since it's much safer. But I am curious how did it pass the unit test previously. Seems the database `default` will not be created while initializing, unless we create it explicitly?